lib/plugins/aws/deploy/lib/mergeIamTemplates.js

```diff
@@ -4,6 +4,19 @@ const _ = require('lodash');
 const BbPromise = require('bluebird');
 const path = require('path');
 
+function checkStatements(statements) {
```

Comment:
One more thing: I had a couple of commits which I forgot to "git pull" before you reviewed... 
so you're going to see the following changes (in addition to the changes you requested):

a/ checkStatements() was renamed to validateStatements() and it now either returns gracefully or throws an error (so that caller does not have to deal with checking the return value and throwing an error off of it)
b/ slight changes to the phrasing of the error message
c/ a few more unit tests